### DISCUSSION TITLE
Bad slicing with new versions of Cura

### DISCUSSION THREAD

**Discussion by Cesinha1979** [community/community#27753] [Actions]
![Cura - comparativo](https://user-images.githubusercontent.com/110311768/182003736-ff800d4d-2fd6-418b-b240-93e5f2744998.jpg)

I Was using version 4.3 and  installed the nem version (5.0 and 5.1) and the slicing is been different. The seams is not in the same point.
Why the seam is so different? 
And it seams the objects are worst now
I did a printscreen to compare:

**Comment by airtower-luna** [comment]
It looks like you're trying to report a bug in a software hosted on GitHub, maybe https://github.com/Ultimaker/Cura? In that case you should open an issue *in that repository*. :slightly_smiling_face:
